distribution/docs/src/main/resources/content/_reference/_appReferences/mg-spatial.adoc

```diff
@@ -47,6 +47,45 @@ index again.
 
 |===
 
+=== Standalone Solr Offline Gazetteer Feature
+
+The Standalone Solr Offline Gazetteer is a reflection of the gazetteer data stored
+in the catalog but separated from other data. This allows the core to be
+separately reindexed much quicker than the the entire catalog core.
+
+====  Installing Standalone Solr Offline Gazetteer
+
+The Standalone Solr Offline Gazetteer is not installed by default but can be installed by running
+`feature:install catalog-solr-offline-gazetteer`. This will install:
+
+* The plugin that intercepts gazetteer metacard creates/updates/deletes to the catalog core
```

Comment:
✏️ Replace mentions to `core` to `collection`.

Reply:
Good catch 👌